Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
Ok, I will merge this now along with `-R` flag in the instruction. And will replace it when we implemented `brew repair-permission` with more fine grained file inspection.